recipes/mar_greenland/recipe.py

```diff
@@ -0,0 +1,14 @@
+from pangeo_forge_recipes.patterns import ConcatDim, FilePattern
+from pangeo_forge_recipes.recipes import XarrayZarrRecipe
+
+ids = list(range(2000, 2022))
+id_concat_dim = ConcatDim('id', ids, nitems_per_file=1)
+
+
+def make_url(id):
+    return f'https://storage.googleapis.com/ldeo-glaciology/MAR/6.5km/MARv3.12-6_5km-daily-ERA5-{id}.nc'
```

Comment:
The linter is complaining about the length of this line. Let's add a `#noqa` to ignore this line. 

```suggestion
    return f'https://storage.googleapis.com/ldeo-glaciology/MAR/6.5km/MARv3.12-6_5km-daily-ERA5-{id}.nc' # noqa
```